easybuild/easyconfigs/o/OpenMM/OpenMM-6.3-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'OpenMM'
+version = '6.3'
+
+homepage = 'http://openmm.org/'
+description = "A high performance toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://github.com/pandegroup/openmm/releases']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.2.3'),
+    ('libxml2', '2.9.2'),
+    ('SWIG', '3.0.8', '-Python-2.7.11'),
```

Comment:
please use `%(pyver)s` here rather than hardocding the Python version in two places; note that this requires EasyBuild v2.7.0 or newer

```python
   ('SWIG', '3.0.8', '-Python-%(pyver)s'),
```